applications/app/templates/serviceWorker.scala.js

```diff
@@ -0,0 +1,63 @@
+@()
+// TODO: Lint
+
+/*eslint quotes: [2, "single"], curly: [2, "multi-line"], strict: 0*/
+/*eslint-env browser*/
+/*global self*/
+// TODO: eslint-env service-worker
+// https://github.com/eslint/eslint/issues/2557
+/*global clients*/
```

Comment:
FYI: I'm starting work in another branch to move us from JSHint + JSCS to ESLint (much more configurable, better ES6 support, built on Esprima). @sndrs already added it for the SystemJS worktree :smile: